internal/adapters/terraform/aws/lambda/adapt_test.go

```diff
@@ -24,9 +24,9 @@ func Test_Adapt(t *testing.T) {
 			terraform: `
 			resource "aws_lambda_function" "example" {
 				filename      = "lambda_function_payload.zip"
-				function_name = "lambda_function_name"
+				function_name = ""
 				role          = aws_iam_role.iam_for_lambda.arn
-				runtime = "nodejs12.x"
+				runtime = ""
```

Comment:
could you explain this change? functions have these fields populated

Reply:
Yes, When I ran test cases it appeared in the logs that actual and expected values are different. in actual value **runtime** coming empty and same case with **fucntion_name** to remediate the failing of test case I changed the expected value here.

Reply:
I don't think that's correct. Why the returned value is empty? is this documented somewhere on AWS API?